docs/src/main/asciidoc/firestore.adoc

```diff
@@ -110,7 +110,7 @@ include::{project-root}/spring-cloud-gcp-data-firestore/src/test/java/org/spring
 This annotation is optional, by default the collection name is derived from the class name.
 
 `@DocumentId` annotation marks a field to be used as document id. This annotation is required.
-If the value is null, and the property type is `String`, the Id is generated automatically.
+NOTE: If the property annotated with `@DocumentId` is `null` and the property type is `String`, the document id is generated automatically when the entity is saved.
```

Comment:
Add a new line above the `NOTE:` so that it gets rendered correctly as a note in the asciidoc.

Reply:
@dmitry-s This is still not addressed. It won't render correctly otherwise.